jinglify/Game/GameSettings.swift

```diff
@@ -30,6 +30,8 @@ struct GameSettings {
             }).flatMap({ (predicate) in
                 let query = MPMediaQuery()
                 query.addFilterPredicate(predicate)
+                query.addFilterPredicate(MPMediaPropertyPredicate(value: false, forProperty: MPMediaItemPropertyHasProtectedAsset))
+                query.addFilterPredicate(MPMediaPropertyPredicate(value: false, forProperty: MPMediaItemPropertyIsCloudItem))
```

Comment:
Because it doesn't have AssetUrl. Apple disallow usage of protected content (i.e. from Apple Music) with AVAudioPlayer.